### DISCUSSION TITLE
客户端跟服务器端的打包编译

### DISCUSSION THREAD

**Discussion by Mrming1994** [DrDao/OpenMir2#37] [General]
求客户端跟服务器端的打包编译流程,最好详细一点

**Comment by mirbeta** [comment]
服务端目前还没办法做到开箱即用，不过可以把你遇到的问题贴出来帮你解决，目前正在努力实现开箱即用。

**Reply by Mrming1994** [reply]
我是一个纯新手, 因为几个朋友想玩传奇1.76, 想从0到1把热血传奇服务器跟客户端都编译运行出来,在网上找了很久的教程都没有答案,非常的迷茫

**Reply by Mrming1994** [reply]
我有一台windows外网服务器,所以想把服务端部署在windows上陪朋友一起玩

**Reply by mirbeta** [reply]
这种情况建议你下载国内其他的的开箱即用引擎...gom leg gee等

**Reply by Mrming1994** [reply]
有没有源码编译的教程给我借鉴一下

**Reply by mirbeta** [reply]
暂时还没有相关教程，编译相对简单一点，IDE环境安装好之后打开项目即可通过编译

**Reply by Mrming1994** [reply]
![微信图片_20230301171542](https://user-images.githubusercontent.com/43674158/222095584-b3c04735-f1ff-4486-8ab2-32b8e86ea0c5.png)
编译报错了
